system-test/datastore.ts

```diff
@@ -305,6 +305,21 @@ describe('Datastore', () => {
       await datastore.delete(postKey);
     });
 
+    it('should save/get/merge', async () => {
+      const postKey = datastore.key(['Post', 1]);
+      await datastore.save({key: postKey, data: post});
+      const entity = {
+        key: postKey,
+        data: {
+          title: 'Updated',
+        },
+      };
+      await datastore.merge(entity);
+      const [entity2] = await datastore.get(postKey);
```

Comment:
it might be good to demonstrate in this system test that original values will be maintained if they're not included in `data`, e.g., that:

```
const obj = {title: 'original', status: 'neat};
const obj2 = {title: 'updated'}
merge(obj, obj2)
// will result inwill result in 
{title: 'updated', status; 'neat'}
```

Reply:
done!